src/core_modules/capture-core/components/FormFields/New/Fields/AgeField/AgeField.component.js

```diff
@@ -49,6 +49,7 @@ const AgeFieldPlain = (props: Props) => {
         // $FlowFixMe[cannot-spread-inexact] automated comment
         <UIAgeField
             datePlaceholder={systemSettingsStore.get().dateFormat.toLowerCase()}
+            locale={systemSettingsStore.get().uiLocale}
```

Comment:
Pushing locale to the dhis2 UI calendar component. Days of the week and months are shown in locale language. Not sure why we didn't do this earlier? It seems to work. Can you verify?